lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
Sounds like a sensible suggestion, @r-obert. However, should it be done in this PR? It sounds like a job for a separate one.